Makefile

```diff
@@ -0,0 +1,24 @@
+all: build
+
+build:
+	cargo build --release
+
+install:
+	cp target/release/wirespider /usr/local/bin
+	if ! [ -e /etc/wirespider/wg0 ]; then \
+		cp systemd/wirespider/wg0 /etc/wirespider/wg0; \
+	else \
+		cp -f systemd/wirespider/wg0 /etc/wirespider/wg0.new; \
+	fi
+	cp target/release/wirespider /usr/local/bin
+	mkdir -p /etc/wirespider/keys
+	cp systemd/system/wirespider-client@.service /etc/systemd/system
```

Comment:
It would be good if a DESTDIR would be used. See here: https://www.gnu.org/prep/standards/html_node/DESTDIR.html